### PR TITLE
MTDSA-5152: Added audits for payments and liabilities

### PR DESCRIPTION
**Pull request by narendravijayarao**

### CODE REVIEW

File: app/v1/controllers/LiabilitiesController.scala [hunk 1]
```diff
@@ -57,6 +60,9 @@ class LiabilitiesController @Inject()(val authService: EnrolmentsAuthService,
         logger.info(message = s"${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] " +
           s"Successfully retrieved Liabilities from DES")
 
+        auditService.auditEvent(AuditEvents.auditReturns(serviceResponse.correlationId,
```
Comment (yangwong9664) [inline app/v1/controllers/LiabilitiesController.scala:63]: should be "auditLiability" not "auditReturns" right?

File: app/v1/controllers/LiabilitiesController.scala [hunk 2]
```diff
@@ -65,6 +71,10 @@ class LiabilitiesController @Inject()(val authService: EnrolmentsAuthService,
       result.leftMap { errorWrapper =>
         val correlationId = getCorrelationId(errorWrapper)
         val result = errorResult(errorWrapper).withApiHeaders(correlationId)
+
+        auditService.auditEvent(AuditEvents.auditReturns(correlationId,
```
Comment (yangwong9664) [inline app/v1/controllers/LiabilitiesController.scala:75]: same here

File: app/v1/controllers/PaymentsController.scala [hunk 3]
```diff
@@ -63,6 +66,9 @@ extends AuthorisedController(cc) with BaseController with Logging {
           logger.info(s"[${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] " +
             s"Successfully retrieved Payments from DES")
 
+          auditService.auditEvent(AuditEvents.auditReturns(serviceResponse.correlationId,
```
Comment (yangwong9664) [inline app/v1/controllers/PaymentsController.scala:69]: auditPayments not auditReturns

File: app/v1/controllers/PaymentsController.scala [hunk 4]
```diff
@@ -71,6 +77,10 @@ extends AuthorisedController(cc) with BaseController with Logging {
       result.leftMap{errorWrapper =>
         val correlationId = getCorrelationId(errorWrapper)
         val result = errorResult(errorWrapper).withApiHeaders(correlationId)
+
+        auditService.auditEvent(AuditEvents.auditReturns(correlationId,
```
Comment (yangwong9664) [inline app/v1/controllers/PaymentsController.scala:81]: same here

File: app/v1/controllers/LiabilitiesController.scala (hunk 1 above)
Comment (narendravijayarao) [reply app/v1/controllers/LiabilitiesController.scala:63]: sorry

File: test/v1/controllers/PaymentsControllerSpec.scala [hunk 5]
```diff
@@ -189,6 +202,10 @@ class PaymentsControllerSpec
             status(result) shouldBe expectedStatus
             contentAsJson(result) shouldBe Json.toJson(mtdError)
             header("X-CorrelationId", result) shouldBe Some(correlationId)
+
+            val auditResponse: AuditResponse = AuditResponse(OK, Some(Seq(AuditError(mtdError.code))), None)
+            MockedAuditService.verifyAuditEvent(AuditEvents.auditReturns(correlationId,
```
Comment (yangwong9664) [inline test/v1/controllers/PaymentsControllerSpec.scala:207]: same for tests

**Pull request merged**